library/acl.inc

```diff
@@ -787,7 +784,7 @@
                         LEFT JOIN module_acl_sections AS  acl_sections
                           ON group_settings.section_id = acl_sections.section_id
                       WHERE
-                        acl_sections.`section_identifier` = ? AND group_settings.group_id IN (".implode('?,',$arr_temp).") AND group_settings.allowed = ?";
+                        acl_sections.`section_identifier` = ? AND group_settings.group_id IN (?) AND group_settings.allowed = ?";
```

Comment:
I am pretty sure this won't work across different sql connectors(adodb, pdo, etc), which is important since may change to PDO at some point:
http://stackoverflow.com/questions/920353/can-i-bind-an-array-to-an-in-condition